proto/ignition/msgs/wrench_stamped.proto

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2017 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+syntax = "proto3";
+package ignition.msgs;
+option java_package = "com.ignition.msgs";
+option java_outer_classname = "WrenchStampedProtos";
+
+/// \ingroup ignition.msgs
+/// \interface Wrench Stamped
+/// \brief Message for a wrench value
+
+import "ignition/msgs/wrench.proto";
+import "ignition/msgs/header.proto";
+
+message WrenchStamped
+{
+  /// \brief Optional header data
+  Header header                   = 1;
+
+  int32 entity                    = 2;
+  string plugin                   = 4;
```

Comment:
This field seems very specific to a single use case. It would be interesting to make this more general. Maybe `label`? For example, a single plugin may publish many of these and want to tell them apart.

Reply:
6dde033